pkgs/development/ocaml-modules/gitlab/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, buildDunePackage, fetchFromGitHub
+, uri, cohttp-lwt, atdgen, yojson
+, iso8601, stringext
+}:
+
+buildDunePackage rec {
+  pname = "gitlab";
+  version = "0.1.8";
+
+  src = fetchFromGitHub {
+    owner = "tmcgilchrist";
+    repo = "ocaml-gitlab";
+    rev = "${version}";
+    sha256 = "sha256-7pUpH1SoP4eW8ild5j+Tcy+aTXq0+eSkhKUOXJ6Z30k=";
+  };
+
+  duneVersion = "3";
+  minimalOCamlVersion = "4.08";
+
+  nativeBuildInputs = [ atdgen ];
+
+  propagatedBuildInputs = [
+    uri
+    cohttp-lwt
+    atdgen
+    yojson
+    iso8601
+    stringext
```

Comment:
```suggestion
```

Does `stringext` needs to be propagated?

Reply:
Just confirmed this, it does not need propagation.

Reply:
What about `buildInputs`?

Reply:
Added below.